test/test.spec.js

```diff
@@ -6,8 +6,8 @@
  */
 'use strict';
 
-const path = require('path');
-const should = require('should/as-function');
+const path = require('node:path');
+const assert = require('node:assert');
```

Comment:
replacing "should" with "assert"

Reply:
I recommend using the strict assertion mode:

```suggestion
const assert = require('node:assert/strict');
```

See https://nodejs.org/api/assert.html#strict-assertion-mode